group_project_v2/mixins.py

```diff
@@ -126,15 +128,21 @@ class WorkgroupAwareXBlockMixin(UserAwareXBlockMixin, CourseAwareXBlockMixin):
     """
     Gets current user workgroup, respecting TA review
     """
-    FALLBACK_WORKGROUP = {"id": "0", "users": []}
+    FALLBACK_WORKGROUP = WorkgroupDetails(id="0", users=[])
 
     @property
     def group_id(self):
-        return self.workgroup['id']
+        """
+        :rtype: int
+        """
+        return self.workgroup.id
 
     @property
     def is_group_member(self):
-        return self.user_id in [u["id"] for u in self.workgroup["users"]]
+        """
+        :rtype: bool
+        """
+        return self.user_id in [u.id for u in self.workgroup.users]
```

Comment:
@bradenmacdonald users are already loaded, so the benefit would be rather negligible. I.e. `self.workgroup` is a result of `get_workgroup_by_id`, and the response already contains `ReducedUserDetails` data.